README.md

```diff
@@ -1,14 +1,16 @@
 # GovWifi Logging API
 
-Session logging for successful authentications
+The frontend uses this API to record each session with successful authentications.
+
+The private GovWifi [build repository][build-repo] contains instructions on how to build GovWifi end-to-end - the sites, services and infrastructure.
 
 ## Overview
 
 Also known as post-auth in Freeradius terms, this logs to the sessions table when a user has authenticated successfuly.
```

Comment:
During the RADIUS `post-auth` action a POST request is sent to this API containing session data. This API receives this request and saves it to a database.